File: VB/02_ParagraphAndText/RemoveTextBox.vb
Imports Spire.Presentation

Namespace RemoveTextBox
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
			'Create an instance of presentation document
			Dim ppt As New Presentation()
			'Load file
			ppt.LoadFromFile("..\..\..\..\..\..\Data\TextBoxTemplate.pptx")

			'Get the first slide
			Dim slide As ISlide = ppt.Slides(0)
			'Traverse all the shapes in slide
			Dim i As Integer = 0
			Do While (i < slide.Shapes.Count)
				If slide.Shapes(i).Name.Contains("TextBox") Then
					slide.Shapes.RemoveAt(i)
					i=(i-1)
				End If
				i=(i+1)
			Loop

			'Save the document
			Dim result As String = "RemoveTextBox.pptx"
			ppt.SaveToFile(result, FileFormat.Pptx2013)
			PresentationDocViewer(result)

		End Sub

		Private Sub PresentationDocViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace

File: CS/02_ParagraphAndText/RemoveTextBox.cs
using System;
using System.Windows.Forms;
using Spire.Presentation;

namespace RemoveTextBox
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();            
        }

        private void btnRun_Click(object sender, EventArgs e)
        {
            //Create an instance of presentation document
            Presentation ppt = new Presentation();
            //Load file
            ppt.LoadFromFile(@"..\..\..\..\..\..\Data\TextBoxTemplate.pptx");

            //Get the first slide
            ISlide slide = ppt.Slides[0];
            //Traverse all the shapes in slide
            for (int i = 0; i < slide.Shapes.Count;i++)
            {
                if(slide.Shapes[i].Name.Contains("TextBox"))
                {
                	slide.Shapes.RemoveAt(i);
                	i--;
                }
            }

            //Save the document
            string result = "RemoveTextBox.pptx";
            ppt.SaveToFile(result, FileFormat.Pptx2013);
            PresentationDocViewer(result);
        
	    }
	
	    private void PresentationDocViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}